Source/iop/Iop_Dmac.h

```diff
@@ -31,10 +31,10 @@ namespace Iop
 			CH4_BASE = 0x1F8010C0,
 			CH5_BASE = 0x1F8010D0,
 			CH6_BASE = 0x1F8010E0,
-			CH8_BASE = 0x1F801500,
-			CH9_BASE = 0x1F801510,
-			CH11_BASE = 0x1F801530, //Unsure about that
-			CH12_BASE = 0x1F801540, //Unsure about that
+			CH7_BASE = 0x1F801500,
+			CH8_BASE = 0x1F801510,
+			CH11_BASE = 0x1F801540, //Unsure about that

```

Comment:
We should get rid of the comments that say we're unsure :)

Reply:
Done :D